### ISSUE TITLE
Feature Request: Annotations on selected frames

### ISSUE THREAD

**Issue by bobhenkel** [firatkiral/pypeek#12]
I love this app!

Would it be possible to put annotations elements on only select frames? For example, sometimes I want to record my web browser app and I have to move from one tab to another in the web app. When on the first tab I may circle something in red I don't want that circle to show up through out the recording because I will show things on the second tab and the thing I circle is no longer visible making the red circle out of place.

**Comment by jkcapoeira** [comment]
Yes that would be great! I am also interested in that functionality to make instruction video's. Now using wink sometimes, but that is old and not stable.

**Comment by kintaro1981** [comment]
Yes for tutorial videos it is extremely usefull the possibility to annotate only on some frames and other annotations on other frames
